src/main/java/broski/taskRelated/TaskManager.java

```diff
@@ -0,0 +1,110 @@
+package broski.taskRelated;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import broski.task.Deadline;
+import broski.task.Event;
+import broski.task.Task;
+import broski.task.Todo;
+
+/**
+ * Class that holds all functionality for the saving and loading of tasks into hard disk.
+ */
+public class TaskManager {
+
+    private static final String FILE_PATH = "./data/broski.txt";
```

Comment:
I like that all constants are named correctly. Excellent!

Reply:
Agreed! It's great that you used SCREAMING_SNAKE_CASE for the constant